pkg/master/master.go

```diff
@@ -87,22 +88,25 @@ const (
 
 // Config is a structure used to configure a Master.
 type Config struct {
-	DatabaseStorage tools.StorageInterface
-	EventTTL        time.Duration
-	MinionRegexp    string
-	KubeletClient   client.KubeletClient
+	DatabaseStorage    tools.StorageInterface
+	ExpDatabaseStorage tools.StorageInterface
```

Comment:
Different codecs if you don't share a scheme

On Jul 29, 2015, at 9:34 PM, Nikhil Jindal <notifications@github.com> wrote:

In pkg/master/master.go
<https://github.com/GoogleCloudPlatform/kubernetes/pull/12001#discussion_r35831210>
:

> @@ -87,22 +88,25 @@ const (
>
>  // Config is a structure used to configure a Master.
>  type Config struct {
> -	DatabaseStorage tools.StorageInterface
> -	EventTTL        time.Duration
> -	MinionRegexp    string
> -	KubeletClient   client.KubeletClient
> +	DatabaseStorage    tools.StorageInterface
> +	ExpDatabaseStorage tools.StorageInterface

Clarification: We can use the same etcdStorage as well, right?

—
Reply to this email directly or view it on GitHub
<https://github.com/GoogleCloudPlatform/kubernetes/pull/12001/files#r35831210>
.